src/content/docs/cheerpj-applet-runner/03-getting-started/using-first-time.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Using the extension for the first time
+description: Tutorial for first-time users
+---
+
+This tutorial will guide you on the basic functionalities of the Applet Runner extension. We assume you have installed the extension on [Chrome](/cheerpj-applet-runner/getting-started/install-chrome) or [Edge](/cheerpj-applet-runner/getting-started/install-edge).
+
+<div class="mx-24">![](/cheerpj3/assets/applet-screenshots/homescreen.png)</div>
+
+### How to activate
+
+Visit a page with a Java applet, such as this one and click on the CheerpJ Applet Runner icon in the toolbar and enable CheerpJ.
```

Comment:
There is no link on "such as this one". May be good idea to add a screenshot of the activation dialogue as well.